subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -326,11 +332,12 @@ enum {
 	EVENT_DONE_EXTRA_TYPE_CONN,
 #endif /* CONFIG_BT_CONN */
 
-#if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_CTLR_ADV_EXT) || \
+	defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
```

Comment:
Now that lines can be up to 100 characters, I think this should go onto a single line (also other instances)